Resources/public/js/views/services/ez-subitemboxviewservice.js

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-subitemboxviewservice', function (Y) {
+    "use strict";
+    /**
+     * Provides the view service component for the location view
+     *
+     * @module ez-subitemboxviewservice
+     */
+    Y.namespace('eZ');
+
+    /**
+     * Subitem box view service.
+     *
+     * Loads the models needed by the subitem box view
+     *
+     * @namespace eZ
+     * @class subitemBoxViewService
+     * @constructor
+     * @extends eZ.ViewService
+     */
+    Y.eZ.SubitemBoxViewService = Y.Base.create('subitemBoxViewService', Y.eZ.ViewService, [], {
+        /**
+         * Loads the location, the content and the path for the location id
+         * available in the request and calls the next callback once it's done.
+         *
+         * @method _load
+         * @protected
+         * @param {Function} next
+         */
+        _load: function (next) {
+            var request = this.get('request'),
+                service = this,
+                tasks,
+                endLoad;
+
+            tasks = new Y.Parallel();
+            endLoad = tasks.add();
+            this.search.findContent({
+                viewName: 'search-',
```

Comment:
that's a weird name. Even if it's not really used (yet) in the REST API implementation, you should probably use a meaning full name, something like `location-<LocationId>`